modules/tide_oauth/src/Commands/TideOauthCommands.php

```diff
@@ -33,23 +33,21 @@ public function __construct(EnvKeyGenerator $env_key_generator) {
   /**
    * Generate OAuth keys from Environment variables.
    *
-   * @usage drush tide-oauth:env-keygen
+   * @usage drush tide-oauth:keygen
    *   Generate OAuth keys from Environment variables.
    *
-   * @command tide-oauth:env-keygen
+   * @command tide-oauth:keygen
    * @validate-module-enabled tide_oauth
-   * @aliases toenvkg,tide-oauth-env-keygen
+   * @aliases tokgn,tide-oauth-keygen
    */
   public function generateKeys() {
-    if ($this->envKeyGenerator->hasEnvKeys()) {
-      if ($this->envKeyGenerator->generateEnvKeys()) {
-        // Update Simple OAuth settings.
-        $this->envKeyGenerator->setSimpleOauthKeySettings();
-        $this->io()->success('OAuth keys have been created from TIDE_OAUTH_*_KEY environment variables.');
-      }
-      else {
-        $this->io()->error('Could not generate OAuth keys from TIDE_OAUTH_*_KEY environment variables.');
-      }
+    if ($this->envKeyGenerator->generateEnvKeys()) {
+      // Update Simple OAuth settings.
+      $this->envKeyGenerator->setSimpleOauthKeySettings();
+      $this->io()->success('OAuth keys have been created from TIDE_OAUTH_*_KEY environment variables.');
```

Comment:
@MdNadimHossain @anthony-malkoun this info message may be misleading now, given that the keys may be generated not from the env.

Reply:
@sonnykt should I give a more general one like this - "OAuth keys have been created" ?